CriteoPublisherSdk/Tests/Utility/Checkers/CR_CreativeViewChecker.h

```diff
@@ -36,15 +38,15 @@
 @property(strong, nonatomic, readonly) XCTestExpectation *adCreativeRenderedExpectation;
 @property(strong, nonatomic, readonly) CRBannerView *bannerView;
 @property(strong, nonatomic, readonly) CRInterstitial *interstitial;
-@property(strong, nonatomic, readonly) CRAdUnit *adUnit;
+@property(strong, nonatomic) CRAdUnit *adUnit;
```

Comment:
Why can't this stay RO ? Isn't it working by putting an explicit _adUnit = nil in the ctor ?

Reply:
Updated to keep the property RO